src/java/winstone/Launcher.java

```diff
@@ -352,6 +352,7 @@ public static void main(String argv[]) throws IOException {
             new Launcher(args);
         } catch (Throwable err) {
             Logger.log(Logger.ERROR, RESOURCES, "Launcher.ContainerStartupError", err);
+            System.exit(1);
```

Comment:
Unrelated, but OK. Maybe deserves a separate changelog entry if there is a real issue with it